client/package.json

```diff
@@ -233,6 +239,14 @@
                 {
                     "command": "blockchainExplorer.restartFabricRuntime",
                     "when": "view == blockchainExplorer && viewItem == blockchain-runtime-item-started"
+                },
+                {
+                    "command": "blockchainExplorer.toggleFabricRuntimeDevMode",
+                    "when": "view == blockchainExplorer && viewItem == blockchain-runtime-item-started"
```

Comment:
Can't you just have an or here rather than adding it twice

Reply:
Nope; already tried that with braces surrounding the or condition; it did not work.

Reply:
ok you are right it doesn't work :(